duckduckgo/Android
# Bump minor libraries to latest version

Task/Issue URL: https://app.asana.com/0/414730916066338/1199111180919669/f
Tech Design URL: 
CC: 

**Description**:
This PR updates some minor libraries to the latest stable version available.

Libraries included here:
- testRunner
- leak canary
- mockito
- mockito kotlin

We've also removed all dependencies related to `testImplementation` as they are not relevant. We only have `androidTest` in our codebase.

**Steps to test this PR**:
1. Run all test, they should pass without errors.

1. Install the app and ensure leak canary works for debug builds.


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
